inst/www/shinyTable.js

```diff
@@ -320,10 +320,9 @@ Shiny.addCustomMessageHandler('htable-change', function(data) {
   
   var tbl = $el.handsontable('getInstance');
   for( var i = 0; i < data.changes.length; i++){
-    var change = data.changes[i];
     tbl.setDataAtCell(
-      parseInt(change.row), 
-      parseInt(change.col),
+      parseInt(data.changes.row[i]), 
+      parseInt(data.changes.col[i]),
       change.new,
```

Comment:
Now references a variable that was deleted. Seems like the right fix is to add the var change line back in and just add `[i`] to lines 324 and 325 and otherwise leave them as they were, no?